app/views/ansible_roles/import.html.erb

```diff
@@ -0,0 +1,49 @@
+<% title _("Changed roles") %>
+<%= form_tag confirm_import_ansible_roles_path do %>
+  <h4><%= _("Select the changes you want to realize in Foreman") %></h4>
+  <h6>
+  <%= _("Toggle") %>:
+  <%= link_to_function(icon_text("check", _("New")),
+                       "toggleCheckboxesBySelector('.role_select_boxes_new')",
+                       :title => _("Check/Uncheck new")) %> |
+  <%= link_to_function(icon_text("check", _("Obsolete")),
+                       "toggleCheckboxesBySelector('.role_select_boxes_obsolete')",
+                       :title => _("Check/Uncheck obsolete")) %>
+  </h6>
+  <table class="<%= table_css_classes %>">
+    <thead>
+      <tr>
+        <th class="ca">
+          <%= link_to_function(icon_text("check"),
+                               "toggleCheckboxesBySelector('.role_select_boxes')",
+                               :title => _("Check/Uncheck all")) %>
+        </th>
+        <th><%= _("Name") %></th>
+        <th><%= _("Operation") %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% ["new", "obsolete"].each do |kind| %>
```

Comment:
This should show something when no new or obsolete roles were found, right now it just displays an empty table which is kind of confusing. I think at that point it'd make sense to take the tbody and put it in a partial